io.github.tofudd.lanchat.terminal.metainfo.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be ideally maintained in your repository and not on Flathub; this way you can update it as part of the release workflow.